mim install mmaction2 error

```
mim install mmaction2
installing mmaction2 from https://github.com/open-mmlab/mmaction2.git.
Cloning into '/tmp/tmpabjk9q6z/mmaction2'...
remote: Enumerating objects: 14462, done.
remote: Counting objects: 100% (435/435), done.
remote: Compressing objects: 100% (325/325), done.
remote: Total 14462 (delta 225), reused 202 (delta 110), pack-reused 14027
Receiving objects: 100% (14462/14462), 48.65 MiB | 10.33 MiB/s, done.
Resolving deltas: 100% (10490/10490), done.
Note: switching to '48ba0fb6dbceb0084f95f4ec288b23b244d2c360'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

installing dependency: mmcv-full
installing mmcv-full from wheel.
Looking in links: https://download.openmmlab.com/mmcv/dist/cu113/torch1.10.0/index.html
Collecting mmcv-full==1.4.1
  Using cached https://download.openmmlab.com/mmcv/dist/cu113/torch1.10.0/mmcv_full-1.4.1-cp39-cp39-manylinux1_x86_64.whl (58.1 MB)
Requirement already satisfied: numpy in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from mmcv-full==1.4.1) (1.21.2)
Requirement already satisfied: pyyaml in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from mmcv-full==1.4.1) (6.0)
Requirement already satisfied: Pillow in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from mmcv-full==1.4.1) (8.4.0)
Collecting yapf
  Using cached yapf-0.31.0-py2.py3-none-any.whl (185 kB)
Collecting packaging
  Using cached packaging-21.3-py3-none-any.whl (40 kB)
Collecting addict
  Using cached addict-2.4.0-py3-none-any.whl (3.8 kB)
Collecting opencv-python>=3
  Using cached opencv_python-4.5.4.60-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (60.3 MB)
Collecting pyparsing!=3.0.5,>=2.0.2
  Using cached pyparsing-3.0.6-py3-none-any.whl (97 kB)
Installing collected packages: pyparsing, yapf, packaging, opencv-python, addict, mmcv-full
Successfully installed addict-2.4.0 mmcv-full-1.4.1 opencv-python-4.5.4.60 packaging-21.3 pyparsing-3.0.6 yapf-0.31.0
Successfully installed mmcv-full.
Successfully installed dependencies.
Requirement already satisfied: numpy in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from -r /tmp/tmpabjk9q6z/mmaction2/requirements/build.txt (line 2)) (1.21.2)
Requirement already satisfied: torch>=1.3 in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from -r /tmp/tmpabjk9q6z/mmaction2/requirements/build.txt (line 3)) (1.10.1)
Requirement already satisfied: typing_extensions in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from torch>=1.3->-r /tmp/tmpabjk9q6z/mmaction2/requirements/build.txt (line 3)) (3.10.0.2)
Processing /tmp/tmpabjk9q6z/mmaction2
Collecting decord
  Using cached decord-0.6.0-py3-none-manylinux2010_x86_64.whl (13.6 MB)
Collecting einops
  Using cached einops-0.3.2-py3-none-any.whl (25 kB)
Collecting matplotlib
  Using cached matplotlib-3.5.1-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.whl (11.2 MB)
Requirement already satisfied: numpy in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from mmaction2==0.20.0) (1.21.2)
Collecting opencv-contrib-python
  Using cached opencv_contrib_python-4.5.4.60-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (66.4 MB)
Requirement already satisfied: Pillow in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from mmaction2==0.20.0) (8.4.0)
Collecting scipy
  Using cached scipy-1.7.3-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (39.8 MB)
Requirement already satisfied: packaging>=20.0 in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from matplotlib->mmaction2==0.20.0) (21.3)
Collecting fonttools>=4.22.0
  Using cached fonttools-4.28.4-py3-none-any.whl (885 kB)
Requirement already satisfied: pyparsing>=2.2.1 in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from matplotlib->mmaction2==0.20.0) (3.0.6)
Collecting cycler>=0.10
  Using cached cycler-0.11.0-py3-none-any.whl (6.4 kB)
Collecting kiwisolver>=1.0.1
  Using cached kiwisolver-1.3.2-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (1.6 MB)
Requirement already satisfied: python-dateutil>=2.7 in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from matplotlib->mmaction2==0.20.0) (2.8.2)
Requirement already satisfied: six>=1.5 in /home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages (from python-dateutil>=2.7->matplotlib->mmaction2==0.20.0) (1.16.0)
Building wheels for collected packages: mmaction2
  Building wheel for mmaction2 (setup.py) ... done
  Created wheel for mmaction2: filename=mmaction2-0.20.0-py2.py3-none-any.whl size=732059 sha256=a748ee6a81bbfcd656580e7bdc7d776a71fbb2c2c53c033ca93ad831aba0a0ca
  Stored in directory: /tmp/pip-ephem-wheel-cache-gjm979m4/wheels/88/80/50/872e884bbe3b65aa2544afe4b28753b178b145678fa4cc2b39
Successfully built mmaction2
Installing collected packages: kiwisolver, fonttools, cycler, scipy, opencv-contrib-python, matplotlib, einops, decord, mmaction2
Successfully installed cycler-0.11.0 decord-0.6.0 einops-0.3.2 fonttools-4.28.4 kiwisolver-1.3.2 matplotlib-3.5.1 mmaction2-0.20.0 opencv-contrib-python-4.5.4.60 scipy-1.7.3
Traceback (most recent call last):
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/pkg_resources/__init__.py", line 344, in get_provider
    module = sys.modules[moduleOrReq]
KeyError: 'mmaction'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/louis/miniconda3/envs/open-mmlab/bin/mim", line 8, in <module>
    sys.exit(cli())
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/mim/commands/install.py", line 101, in cli
    install(
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/mim/commands/install.py", line 205, in install
    install_from_github(target_pkg, target_version, find_url, timeout,
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/mim/commands/install.py", line 536, in install_from_github
    install_from_repo(
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/mim/commands/install.py", line 498, in install_from_repo
    copy_file_to_package()
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/mim/commands/install.py", line 412, in copy_file_to_package
    mim_root = resource_filename(module_name, '.mim')
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/pkg_resources/__init__.py", line 1130, in resource_filename
    return get_provider(package_or_requirement).get_resource_filename(
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/pkg_resources/__init__.py", line 346, in get_provider
    __import__(moduleOrReq)
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/mmaction/__init__.py", line 11, in <module>
    assert (digit_version(mmcv_minimum_version) <= mmcv_version
AssertionError: MMCV==1.4.1 is used but incompatible. Please install mmcv>=1.3.6, <=1.4.0.
```


It's clear the problem comes from the wrong version of `mmcv`. But since `mim` is an automated installation helper, it's should check and use the compatible mmcv by itself.

I checked the __init__.py in `mmaction`, it sets the max version of `mmcv==1.5.0`:
https://github.com/open-mmlab/mmaction2/blob/9ff6a14c103967d97e193957370392c4acff9936/mmaction/__init__.py#L8
But the error raised say it is `1.4.0`. Should be a bug? Maye be I should post this issue in `openmim` repo?

--- a/docs/install.md
+++ b/docs/install.md
@@ -92,7 +92,7 @@ We recommend you to install MMAction2 with [MIM](https://github.com/open-mmlab/m
 
 ```shell
 pip install git+https://github.com/open-mmlab/mim.git
-mim install mmaction2
+mim install mmaction2 -f https://github.com/open-mmlab/mmaction2.git
 ```
 
 MIM can automatically install OpenMMLab projects and their requirements.

--- a/docs_zh_CN/install.md
+++ b/docs_zh_CN/install.md
@@ -87,7 +87,7 @@ conda install pytorch=1.3.1 cudatoolkit=9.2 torchvision=0.4.2 -c pytorch
 
 ```shell
 pip install git+https://github.com/open-mmlab/mim.git
-mim install mmaction2
+mim install mmaction2 -f https://github.com/open-mmlab/mmaction2.git
 ```
 
 MIM 可以自动安装 OpenMMLab 项目及其依赖。

